magma.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module version="4">
+  <component name="Go" enabled="true" />
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```

Comment:
@themarwhal I think this gets created randomly also (happened in a different PR I opened and closed)
Might be related to what you said before

> @markjen / @hcgatewood I'm not sure why this keeps happening, but some unrelated Golang development seems to affect this file.

Reply:
Ye will add to .gitignore